examples/js/loaders/FBXLoader.js

```diff
@@ -344,8 +364,18 @@
 
 		}
 
-		var texture = loader.load( fileName );
-
+		var texture;
+	
+		if (textureNode.FileName.slice(textureNode.FileName.lastIndexOf('.') + 1)=='tga'){
```

Comment:
Need spaces to follow the code style. And required to use `===` instead of `==`.

    if ( textureNode.FileName.slice( textureNode.FileName.lastIndexOf( '.' ) + 1 ) === 'tga' ) {

Reply:
Hi @takahirox,

thanks for suggestions. I updated the PR style. Anyway I still don't know if it is the right way to check if this is fbx file. It works but maybe there is better way?

Reply:
This looks OK, but you need to account for capitalised file extensions like "TGA".

Reply:
What about  `if ( textureNode.FileName.slice( -3 ).toLowerCase() === 'tga' )` ?

Reply:
You're right @looeee. I updated the PR.

Reply:
IMO using `.lastIndexOf( '.' )` would be more robust. For example, `if ( textureNode.FileName.slice( -3 ).toLowerCase() === 'tga' )` can be match with like 'something.atga'.

I haven't looked into `FBXLoader` recently. Is `textureNode.FileName` guaranteed to be correct here? If so, `.slice( -3 )` could be ok.

Reply:
Wouldn't find `jpeg` though...

Reply:
We had an easier solution `textureNode.FileName.slice( -4 ).toLowerCase() === '.tga'`, anyways.